PyPDF2/pdf.py

```diff
@@ -1854,6 +1854,35 @@ def read(self, stream):
             if line[:9] != b_("startxref"):
                 raise PdfReadError("startxref not found")
 
+        #check and eventually correct the startxref only in not strict
+        rebuildXrefTable = False
+        try:
+            stream.seek(startxref - 1,0) #-1 to check character before
+            line=stream.read(1)
+            if line not in b_("\r\n \t"):
```

Comment:
I would expect `x` as in `xref`. Why do you check fir a whitespace?

Reply:
uuuh, wait, do you want to ensure that it's not by accient xref, e.g. within a text string? Might this cause other issues? Is it actually guaranteed that there has to be a whitespace?

Reply:
wanted to ensure that the xref is a word not something else